src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,16 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
+    private final UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+
+        if (user == null) {
+            return false;
+        }
```

Comment:
try to add this check like an additional condition after return, and it will look nice have only one return in this method

Reply:
I don't know how to add a check after the return but I made it in one return